src/main/steps/la-portal/check-your-answers/utils.ts

```diff
@@ -224,7 +224,9 @@ function prepareSummaryList(
           },
         ]
       : []),
-    ...(prefix === FieldPrefix.BIRTH_MOTHER || userCase.birthFatherNameOnCertificate === YesOrNo.YES
+    ...(prefix === FieldPrefix.BIRTH_MOTHER ||
+    userCase.birthFatherNameOnCertificate === YesOrNo.YES ||
+    userCase.birthFatherIdentityKnown === YesOrNo.YES
       ? fieldPrefixBirthMother(
```

Comment:
wonder if this function should be renamed? sounds like it's not just the birth mother fields, but also the father?

Reply:
Unable to rename function due to linting (function has 8 args instead of 7) so added comment instead.